scripts/move-built-files.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -
```

Comment:
A version^1 of what we do in https://github.com/chromaui/blog/blob/master/scripts/move-to-build.sh, which is based on: https://answers.netlify.com/t/support-guide-can-i-deploy-multiple-repositories-in-a-single-site/179

[1] This is unfortunately much more complex, due to the fact that we're (ultimately) moving `/public/*` into `/public/docs/<version>`, but `/public/docs` already exists and its contents need preserved.

Reply:
@winkerVSbecks — This technique did not work. Instead, it results in serving the index page from the `/docs/<version>` route. You can see that here: https://release-6-4--storybook-frontpage.netlify.app/docs/6.4/

Maybe on versioned branch builds I could update the `netlify.toml` ([docs](https://docs.netlify.com/configure-builds/file-based-configuration/#build-settings)) to serve the site from `/public/docs/<version>`?

Reply:
That's true. Yea it won't work cause we're not serving the entire app from that path. 🤦🏽  Let's talk about this tomorrow AM and come up with another plan.

Reply:
Had a thought, so I tried this: https://github.com/storybookjs/frontpage/pull/310/commits/55cfa3ec9ca3ee42763c349a1d5c8e0910b3a006

Interestingly, this (a deploy built from that commit) works: https://61847383eb7e4a00077f7ee6--storybook-frontpage.netlify.app/docs/6.4/docs/6.4/react/get-started/introduction/

Even more interestingly, this (a deploy built from _before_ that commit) **also** works: https://6184680172dea0352ffd920a--storybook-frontpage.netlify.app/docs/6.4/docs/6.4/react/get-started/introduction/

_Mostly capturing these thoughts now, so we can reference in the morning..._